ui/scripts/inlineHtml/inlineScripts.js

```diff
@@ -1044,6 +1047,41 @@ on('remove:repeating_skill',
     updateTotalCP('skill');
   });
 
+on('clicked:addbaseskills', () => {
+  const section = 'skill';
+  const newAttributes = {};
+  function addNewAttributeRow(attributesBySuffix) {
+    const rowId = generateRowID();
+    Object.keys(attributesBySuffix).forEach((attributeName) => {
+      newAttributes[`repeating_${section}_${rowId}_${attributeName}`] = attributesBySuffix[attributeName];
+    });
+  }
+  function addNewDiscipline(skillname, props) {
+    addNewAttributeRow({
+      skilldisciplineinfo: 'D',
+      skillname,
+      ...props,
+    });
+  }
+  function addNewSkill(skillname, props) {
+    addNewAttributeRow({
+      skilldisciplineinfo: '⇡',
+      skillname,
+      ...props,
+    });
+  }
+  addNewDiscipline('People');
+  addNewSkill('Language(Common)', { skillattribute: 'IQ', skillexpertise: 'ST' });
+  addNewSkill('Persuade', { skillattribute: 'IQ', skillexpertise: 'ST' });
+  addNewSkill('People Insight', { skillattribute: 'IQ', skillexpertise: 'ST' });
+  addNewDiscipline('Defense', { skillexpertise: 1 });
+  addNewSkill('Dodge', { skillattribute: 'IQ+DX', skillexpertise: '1', skillbase: -3 });
+  addNewDiscipline('Attack', { skillexpertise: 1 });
+  addNewSkill('Your Weapon', { skillattribute: 'DX', skillexpertise: '1', skillbase: -1 });
+  setAttrs(newAttributes, () => {
+    console.log('Added attributes. The sheet auto-calculates after this.');
+  });
+});
```

Comment:
I also envision a "starting mode screen"
This would show up prior to seeing all the tabs, and would look like:
h1 - What do you want to be?
button - Generate a base character
button - Generate a fighter
button - Generate a mage
button - Generate an innate caster